README.md

```diff
@@ -94,6 +94,26 @@ Note that log messages from other sources such as react prop types will still be
 
 If you want to collect coverage locally, run `npm test -- --coverage`
 
+### Debugging Unit Tests
+Unit tests can be debugged by running jest with the `--inspect-brk` flag and attaching to the node process in vscode's debugger.
+
+- Create or modify the `.vscode/launch.json` config file by adding the following launch config:
+
+   ```jsonc
+   {
+      "type": "node",
+      "request": "attach",
+      "name": "Attach",
+      "port": 9229
+   }
+   ```
```

Comment:
You can add this to `.vscode/settings.json` and commit it. Then it's available to everyone w/o any extra setup. We have a launch chrome w/ debugger attached config in there already

Reply:
It looks like `launch.settings` is gitignored, so I don't think there's a good way to add it that gets committed.